docker/app/models/encoder.py

```
@@ -0,0 +1,117 @@
+        self.dropout2 = nn.Dropout(dropout)
+
+    def forward(self, x, mask=None):
+        """
+        Args:
+            x: 入力テンソル [batch_size, seq_len, d_model]
+            mask: パディングマスク [batch_size, 1, seq_len] または [batch_size, 1, 1, seq_len]
+
+        Returns:
+            出力テンソル [batch_size, seq_len, d_model]
+        """
+        # マスクの次元を調整（必要な場合）
```

Ruff (0.15.11)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)